gitprovider/enums.go

```diff
@@ -182,4 +182,7 @@ const (
 
 	// MergeMethodSquash causes a pull request merge to first squash commits
 	MergeMethodSquash = MergeMethod("squash")
+
+	// MergeMethodRebase causes a pull request merge to first rebase commits
+	MergeMethodRebase = MergeMethod("rebase")
```

Comment:
is supported/tested across all provider implementations?

Reply:
nope.. I'll revert that